doc/quick_start/getting_started_with_Galactic_Ubuntu20.04_ov2.0.md

```diff
@@ -0,0 +1,107 @@
+# ROS2_GALACTIC_OpenVINO_Toolkit
+
+**NOTE:**
+Below steps have been tested on **Ubuntu 20.04**.
+
+## 1. Environment Setup
+* Install ROS2 Galactic ([guide](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html))
+* Install Intel® OpenVINO™ Toolkit Version: 2022.1 ([guide](https://docs.openvino.ai/2022.1/openvino_docs_install_guides_installing_openvino_linux.html)) or building by source code ([guide](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux))
```

Comment:
Why the OpenVINO™ Toolkit Version isn't the directory as below:
 ./openvino/thirdparty/open_model_zoo/tools/model_tools
But source code the directory is existent.

Reply:
The model_tools could be found in sourcecode or after install development_tools.